src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -265,7 +265,9 @@ def _populate_core_stats_and_sim_state(self, area: "Area") -> None:
         core_stats_dict = {"bids": [], "offers": [], "trades": [], "market_fee": 0.0}
 
         if area.current_market:
-            core_stats_dict.update(self._read_market_stats_to_dict(area.current_market))
+            if not ConstSettings.ForwardMarketSettings.ENABLE_FORWARD_MARKETS:
+                # Spot market cannot operate in parallel with the forward markets
+                core_stats_dict.update(self._read_market_stats_to_dict(area.current_market))
```

Comment:
I've actually already created a ticket (GSYE-406) after a conversation with Amir in a different PR :) You can check if that can also work, otherwise feel free to edit it and add more info!